app/controllers/documentController.js

```diff
@@ -1,68 +1,115 @@
-const models = require('../models/dbconnect'),
-  jwt = require('jsonwebtoken');
+'use strict';
```

Comment:
**Nitpick**
Would you mind wrapping your file in a closure? This way, you don't corrupt the global namespace. Effect the change across your codebase

```javascript
(() => {
'use strict'

// Your code goes here

})();